CoreScriptsRoot/ServerCoreScripts/ClientChat/ChatChannel.lua

```diff
@@ -0,0 +1,163 @@
+local source = [[
+local module = {}
+--////////////////////////////// Include
+--//////////////////////////////////////
+local modulesFolder = script.Parent
+local moduleTransparencyTweener = require(modulesFolder:WaitForChild("TransparencyTweener"))
```

Comment:
It's for tweening groups of objects as a single unit, not for generic tweening of single objects.